sample-code/src/index.ts

```diff
@@ -31,7 +31,8 @@ export {
 } from './langchain-azure-openai.js';
 export {
   invokeChain as orchestrationInvokeChain,
-  invokeLangGraphChain
+  invokeLangGraphChain,
+  streamOrchestrationLangChain
```

Comment:
Not necessarily important for now, but I feel like the naming is diverging here a little bit.

[req] I think my idea as to have <type><function><Whatever-chain> as format.

So invokeChain (no type since it's the only variant) but orchestratioInvokeChain (since there are duplicates)
I guess naming wise, we should name invokeLangGraphChain to orchestrationInvokeGraphChain, and your new function to orchestrationStreamChain, what do your think?